lib/watir/cookies.rb

```diff
@@ -59,7 +59,10 @@ def add(name, value, opts = {})
         value: value}
       cookie[:secure] = opts[:secure] if opts.key?(:secure)
       cookie[:path] = opts[:path] if opts.key?(:path)
-      cookie[:expires] = opts[:expires] if opts.key?(:expires)
+      expires = opts[:expires]
+      if expires
+        cookie[:expires] = ::Time.parse(expires) if expires.is_a?(String) else expires
```

Comment:
Prefer the trinary operator for single line conditionals:
`cookie[:expires] = expires.is_a?(String) ? ::Time.parse(expires) : expires`